setup.py

```diff
@@ -76,8 +76,8 @@ def collectExtraFiles():
         "yamlize==0.7.1",
     ],
     extras_require={
-        "mpi": ["mpi4py==3.0.3"],
-        "grids": ["wxpython<=4.1.1"],
+        "mpi": ["mpi4py==3.1.4"],
```

Comment:
I haven't tested this yet, so I think this PR should either remove the version pin to not break requirements downstream or it should hold off until I can test it.